src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -948,4 +951,57 @@ namespace TerminalAppLocalTests
         // will also dismiss itself immediately when that's called. So we can't
         // really inspect the contents of the list in this test, unfortunately.
     }
+
+    void TabTests::TestWindowRenameSuccessful()
+    {
+        auto page = _commonSetup();

```

Comment:
y dis a tab test

Reply:
YEA I know, it doesn't make sense, but I didn't feel like standing up another test class and moving the page instantiation stuff to a common place yet